database/neo4j/neo4j.go

```diff
@@ -0,0 +1,202 @@
+package neo4j
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	neturl "net/url"
+	"sync/atomic"
+
+	"github.com/golang-migrate/migrate/v4/database"
+	"github.com/neo4j/neo4j-go-driver/neo4j"
+)
+
+func init() {
+	db := Neo4j{}
+	database.Register("bolt", &db)
```

Comment:
Let's not register `bolt` since there's another bolt db

Reply:
Ok. The url for the connector is `bolt://`, does that matter?